chore: release docs and script updates | FC-12 (#6056)
openedx/openedx-translations: README.rst

using the `openedx-atlas`_ CLI tool to download specific directories of translation files
from openedx-translations.

This repository implements the `OEP-58`_ proposal.

Main and Release branches
*************************

This repository has a main branch in addition to a dedicated branch for every
release. As of May 10th, 2024 the following are the release branches:

- `main branch <https://github.com/openedx/openedx-translations>`_:

  This branch is used for the latest version of Open edX such as
  `Tutor nightly`_, `edx-platform "master" branch`_ and others.

  To translate the latest versions the `open-edx/openedx-translations`_ Transifex
  project should be used.

- `open-release/redwood.master branch <https://github.com/openedx/openedx-translations/tree/open-release/redwood.master>`_:

  This branch is used for the latest version of Open edX such as
  `Tutor Redwood v18`_, `edx-platform "open-release/redwood.master" branch`_
  and others.

> To translate the latest versions the `open-edx/openedx-translations-redwood`_

Reviewer: ```suggestion
To update translations for the Redwood release, the `open-edx/openedx-translations-redwood`_
```

I think this is important because it's not the "latest version", it's the Redwood one. Also, could you indent this as above?

Transifex project should be used.

Tools for repository maintainers
********************************

> This repository contains both GitHub Actions workflows and Makefile programs

Reviewer: Are the GHA workflows defined in https://github.com/openedx/.github ?

Reviewer 2: The workflows are specific to this repo. For example: [`extract-translation-source-files.yml`](https://github.com/openedx/openedx-translations/blob/main/.github/workflows/extract-translation-source-files.yml) pulls from a bunch of repos throughout the org and makes commits to this repo adding source strings. No github actions are needed on other repos for this process to work.

Author: Good note. I've added a link to avoid confusion.

This repository makes an unusual use of GitHub Actions, therefore it's a special noteworthy case.

using the `openedx-atlas`_ CLI tool to download specific directories of translation files
from openedx-translations.

This repository implements the `OEP-58`_ proposal.

Main and Release branches
*************************

This repository has a main branch in addition to a dedicated branch for every
release. As of May 10th, 2024 the following are the release branches:

``main`` branch
===============

This branch is used for the latest version of Open edX such as
`Tutor nightly`_, `edx-platform "master" branch`_ and others.

To translate the latest versions the `open-edx/openedx-translations`_ Transifex
project should be used.

> ``open-release/redwood.master`` branch

Reviewer: Ah sorry I wasn't explicit, I think we shouldn't use named branches anywhere because we are bad at updating things.

```suggestion
``open-release/<release-name>.master`` branch
```

======================================

This branch is used for the latest version of Open edX such as

> `Tutor Redwood v18`_, `edx-platform "open-release/redwood.master" branch`_

Reviewer: ```suggestion
This branch is used for the latest version of the Open edX Release, which will
be a version of Tutor and corresponding branches in tagged repos. For example,
for the Redwood release (June 2024), the branches were:
`Tutor Redwood v18`_, `edx-platform "open-release/redwood.master" branch`_
```

and others.

> To update translations for the Redwood release, the `open-edx/openedx-translations-redwood`_

Reviewer: ```suggestion
To update translations for a named release, find the corresponding named release project in the `Open edX Transifex project <https://app.transifex.com/open-edx/>`_  by searching for the release name (for example, Redwood) in the search box. 
```

using the `openedx-atlas`_ CLI tool to download specific directories of translation files
from openedx-translations.

This repository implements the `OEP-58`_ proposal.

Main and Release branches
*************************

This repository has a main branch in addition to a dedicated branch for every
release. As of May 10th, 2024 the following are the release branches:

``main`` branch
===============

This branch is used for the latest version of Open edX such as
`Tutor nightly`_, `edx-platform "master" branch`_ and others.

To translate the latest versions the `open-edx/openedx-translations`_ Transifex
project should be used.

> ``open-release/<release-name>.master`` branch
> ======================================

Author: ```suggestion
``open-release/<release-name>.master`` branch
=============================================
```

> This branch is used for the latest version of Open edX such as

Reviewer: ```suggestion
```

This branch is used for the latest version of the Open edX Release, which will
be a version of Tutor and corresponding branches in tagged repos. For example,
for the Redwood release (June 2024), the branches were:
`Tutor Redwood v18`_, `edx-platform "open-release/redwood.master" branch`_
and others.

To update translations for a named release, find the corresponding named release project in the `Open edX Transifex project <https://app.transifex.com/open-edx/>`_  by searching for the release name (for example, Redwood) in the search box. 

> Transifex project should be used.

Reviewer: ```suggestion
```
